src/facade/borrow/borrow.ts

```diff
@@ -90,6 +90,10 @@ export class Borrow {
     );
     const collaterals = await this.getCollaterals(getCollateralValueOption);
 
+    if (collaterals.length === 1 && collaterals[0] === null) {
+      return new terra_js_1.Dec(0).toString();
```

Comment:
Can you change it to `new Dec(0).toString()` ? `Dec` is already imported at the top level.

Reply:
Sure